CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# dbt_shopify v0.16.0
+[PR #97](https://github.com/fivetran/dbt_shopify/pull/97) includes the following updates:
+
+## Bug Fixes
+- Removed incremental logic in the following end models:
+  - `shopify__discounts`
+  - `shopify__order_lines`
+  - `shopify__orders`
+  - `shopify__transactions`
+- Incremental strategies were removed from these models due to potential inaccuracies with the `merge` strategy on BigQuery and Databricks. For instance, the `new_vs_repeat` field in `shopify__orders` could produce incorrect results during incremental runs. To ensure consistency, this logic was removed across all warehouses. If the previous incremental functionality was valuable to you, please consider opening a feature request to revisit this approach.
```

Comment:
My understanding was that it was the incremental strategy in general and not limited to `merge`, especially since we had the models in question materializing as tables by default for BQ and Databricks, and delete+insert uses the same unique-key logic. Is this not the case? Unless I am mistaken, I would recommend generalizing it a bit: 

```suggestion
- Incremental strategies were removed from these models due to potential inaccuracies with the incremental strategy. For instance, the `new_vs_repeat` field in `shopify__orders` could produce incorrect results during incremental runs. If the previous incremental functionality was valuable to you, please consider opening a feature request to revisit this approach.
```

Reply:
Ah, I wasn't certain, as the majority of the conversation centered around `merge`.  I've tweaked this a little, let me know if this looks good!